src/Paket.Core/PublicAPI.fs

```diff
@@ -70,10 +70,7 @@ type Dependencies(dependenciesFileName: string) =
     static member Init() = Dependencies.Init(Directory.GetCurrentDirectory())
 
     /// Initialize paket.dependencies file in the given directory
-    static member Init(directory) =  Dependencies.Init(directory,false)
-
-    /// Initialize paket.dependencies file in the given directory
-    static member Init(directory,fromBootstrapper) =
```

Comment:
While technically a breaking change in the public API - I don't think anyone used these weird overloads.